opentimelineio/adapters/cmx_3600.py

```diff
@@ -845,6 +859,27 @@ def get_content_for_track_at_index(self, idx, title):
         return content
 
 
+def _timing_effects(clip):
+    return [
+        fx for fx in clip.effects
+        if isinstance(fx, otio.schema.LinearTimeWarp)
```

Comment:
Should this be otio.schema.TimingEffect to catch other things in the future?

Reply:
We chatted offline; I'll address this!